contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -1639,10 +1639,6 @@ BEGIN
             v_day := substr(v_datestring, 7, 2);
             v_month := substr(v_datestring, 5, 2);
             v_year := substr(v_datestring, 1, 4);
-        ELSE
```

Comment:
Do we need to replace this ELSE statement with something? Like a default value?

Reply:
This is how the function had it before I made the changes in my last pr so im just restoring it to the original version. If it makes sense to keep this change I can put it back